packages/bot/src/handlers/members/GUILD_MEMBER_UPDATE.ts

```diff
@@ -7,5 +7,6 @@ export async function handleGuildMemberUpdate(bot: Bot, data: DiscordGatewayPayl
   const payload = data.d as DiscordGuildMemberUpdate
 
   const user = bot.transformers.user(bot, payload.user)
+  // @ts-expect-error Flags in the update are nullable, while on the member they are be always present
```

Comment:
Why is this ts-expect error needed?

Reply:
~~isn't the explanation sufficent?~~

`DiscordGuildMemberUpdate` has `flags?: <idr the type name>`, while `DiscordMember` (what the transformer accepts) has `flags: <idr the type name>`, this causes a type error and for this reason there is the expect-error